tls/common.go

```diff
@@ -496,6 +496,10 @@ type Config struct {
 	// material from the returned config will be used for session tickets.
 	GetConfigForClient func(*ClientHelloInfo) (*Config, error)
 
+	// TLSCertsOnly is used to cause a client to close the TLS connection
+	// as soon as the server's certificates have been received
+	TLSCertsOnly bool
```

Comment:
Given that this is already in the `Config` struct in the `tls` package, I think we can just call this `CertsOnly`.

Reply:
renamed in https://github.com/zmap/zcrypto/pull/100/commits/9b39b1586248e55f610c5c1fc6ac508d8a318698